apps/user-office-frontend/src/components/proposal/ProposalTableInstrumentScientist.tsx

```diff
@@ -66,12 +66,26 @@ let columns: Column<ProposalViewData>[] = [
     removable: false,
     field: 'rowActionButtons',
   },
-  { title: 'Proposal ID', field: 'proposalId' },
+  {
+    title: 'Proposal ID',
+    field: 'proposalId',
+  },
   {
     title: 'Title',
     field: 'title',
     ...{ width: 'auto' },
   },
+  {
+    render: (principalInvestigator) => {
+      return `${principalInvestigator.principalInvestigator?.lastname}, ${principalInvestigator.principalInvestigator?.firstname}`;

```

Comment:
There is a chance this can be null given the check is it possible that we also have the '-' instead of undefined.

Reply:
Will including ```emptyValue: '-'``` fix this?

Reply:
Yes this can fix it the we can add this as well to just be cautious some null values   `
    render: (principalInvestigator) => {
      if (
        principalInvestigator.principalInvestigator?.lastname &&
        principalInvestigator.principalInvestigator?.firstname
      ) {
        return `${principalInvestigator.principalInvestigator?.lastname}, ${principalInvestigator.principalInvestigator?.firstname}`;
      }

      return '';
    }`